core/src/main/java/com/netflix/conductor/core/events/EventProcessor.java

```diff
@@ -110,16 +112,22 @@ public EventProcessor(ExecutionService executionService, MetadataService metadat
 	}
 	
 	private void refresh() {
-		Set<String> events = metadataService.getEventHandlers().stream().map(eh -> eh.getEvent()).collect(Collectors.toSet());
-		List<ObservableQueue> created = new LinkedList<>();
-		events.stream().forEach(event -> queuesMap.computeIfAbsent(event, s -> {
-			ObservableQueue q = EventQueues.getQueue(event, false);
-			created.add(q);
-			return q;
-		}));
-		if(!created.isEmpty()) {
-			created.stream().filter(q -> q != null).forEach(queue -> listen(queue));	
-		}
+        try {
+            Set<String> events = metadataService.getEventHandlers().stream().map(eh -> eh.getEvent()).collect(
+                Collectors.toSet());
+            List<ObservableQueue> created = new LinkedList<>();
+            events.stream().forEach(event -> queuesMap.computeIfAbsent(event, s -> {
+                ObservableQueue q = EventQueues.getQueue(event, false);
+                created.add(q);
+                return q;
+            }));
+            if (!created.isEmpty()) {
+                created.stream().filter(q -> q != null).forEach(queue -> listen(queue));
+            }
+        } catch (Exception e) {
+            Monitors.error(className, "event");
+            logger.error(e.getMessage(), e);
```

Comment:
Please log better error message here. `logger.error` already logs the exception with its message.

Reply:
ok,thanks